src/logics/Logic.h

```diff
@@ -444,6 +442,7 @@ class Logic {
     PTRef       resolveTerm        (const char* s, vec<PTRef>& args, char** msg);
     // XXX There's a need for non msg giving version
     virtual PTRef       insertTerm         (SymRef sym, vec<PTRef>& terms, char** msg);
+    virtual PTRef       insertTerm         (SymRef sym, vec<PTRef>&& terms) { return insertTerm(sym, terms, nullptr); }
```

Comment:
It looks like insertTerm is not using the msg parameter (in the actual implementation), so maybe we can the parameter from the first signature?